### PR TITLE
23andMe API & google oauth2 - round trip working example w/ sickle cell anemia

### PR DESCRIPTION
**Pull request by dennisbyrne**
This PR adds a new sample app to your code base.  The sample app uses Google's OAuth 2.0 java lib to authenticate into 23andMe, and pull a single genotype for i3003137, a SNP associated with Sickle Cell Anemia.

### CODE REVIEW

**Comment by dennisbyrne** [comment]
I signed it!

**Comment by dennisbyrne** [comment]
It would be nice to get a link to this contribution on this page as well https://developers.google.com/api-client-library/java/google-oauth-java-client/samples .  Any help on how to get that done would be much appreciated ... thanks in advance.

**Comment by dennisbyrne** [comment]
Can someone please review this PR?

**Comment by dennisbyrne** [comment]
Hello, this PR hasn't received an initial review in six months.  Can someone please review this?

**Comment by tswast** [comment]
CC @lesv @garrettjonesgoogle

File: samples/ttam/src/main/java/com/google/api/services/samples/ttam/SimpleExample.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        AuthorizationCodeFlow flow = new AuthorizationCodeFlow.Builder(authorizationHeaderAccessMethod(),
+                                                             httpTransport,
+                                                             jsonFactory,
+                                                             new GenericUrl(TOKEN_SERVER_URL),
+                                                             credentials,
+                                                             this.clientId,
+                                                             AUTHORIZATION_SERVER_URL)
+                                                            .setScopes(SCOPES)
+                                                            .setDataStoreFactory(dataStoreFactory).build();
+        VerificationCodeReceiver receiver = new LocalServerReceiver.Builder().setHost(DOMAIN)
```
Comment (lesv) [inline samples/ttam/src/main/java/com/google/api/services/samples/ttam/SimpleExample.java]: `.setHost()` should be on it's own line.  Don't think DOMAIN is the right way to deploy this either.

**Comment by dennisbyrne** [comment]
Hello @lesv ... I will address each of your code comments but I was curious as to why you prefer https://github.com/GoogleCloudPlatform/community ?  My reasoning for targeting the PR to this repo began with https://developers.google.com/api-client-library/java/google-oauth-java-client/samples , so I made a peer to https://github.com/google/google-oauth-java-client/tree/master/samples/dailymotion-cmdline-sample (which is currently the only sample).  Thank you for looking at my PR, I will address your other comments shortly.

**Comment by lesv** [comment]
I was thinking folks would be more likely to find it on https://cloud.google.com/community/ .  If you'd prefer it here, I'll merge it. (once the changes are done).

**Comment by dennisbyrne** [comment]
https://cloud.google.com/community/ it is then :)  

I'll need 24 hrs to get to this, thanks.

**Pull request closed without merging**